types/index.d.ts

```diff
@@ -14,20 +14,27 @@ declare module 'fastify' {
     helmet: (opts?: HelmetOptions) => typeof helmet
   }
 
-  export interface RouteOptions extends FastifyHelmetRouteOptions {}
+  export interface RouteOptions extends fastifyHelmet.FastifyHelmetRouteOptions {}
 }
 
-export interface FastifyHelmetRouteOptions {
-  helmet?: Omit<FastifyHelmetOptions, 'global'> | false;
-}
+type FastifyHelmetPlugin = FastifyPluginCallback<fastifyHelmet.FastifyHelmetOptions>;
```

Comment:
I think this is wrong. It is not a Callback but Async

```suggestion
type FastifyHelmetPlugin = FastifyPluginAsync<fastifyHelmet.FastifyHelmetOptions>;
```

Reply:
tks for your suggestion, I think you are right

can you please help with the types issue? I already tried but could not success 

```
  types/index.test-d.ts:103:0
  ✖  103:0  Parameter type ContentSecurityPolicy is declared too wide for argument type FastifyPluginAsync<FastifyHelmetOptions, RawServerDefault, FastifyTypeProviderDefault> & ContentSecurityPolicy.  
  ✖  191:0  Parameter type FastifyPluginAsync<FastifyHelmetOptions, RawServerDefault, FastifyTypeProviderDefault> & { ...; } is declared too wide for argument type typeof fastifyHelmet.    
```